youtube_dl/extractor/generic.py

```diff
@@ -2978,6 +2988,14 @@ def merge_dicts(dict1, dict2):
                     merged[k] = v
             return merged
 
+        # Look for Share-Videos.se embeds
+        sharevideosse_urls = [m.group('url') for m in re.finditer(
+            r'<iframe[^>]+?src=(["\']+)(?P<url>https?://embed\.share-videos\.se/auto/embed/[^"\']+)\1',
```

Comment:
Incorrect quote matching. Expression in quotes may contain the other quote kind characters.

Reply:
Like what characters?

Reply:
What?

Reply:
I mean what characters could match?

Reply:
Attribute value starts with some quote character and ends with exactly this character. Everything in between those quotes may contain any character including other kinds of quotes. What's not clear?

Reply:
Please review my latest commit. Is it correct?